sql/moz-fx-data-shared-prod/search_terms_derived/search_terms_daily_v1/metadata.yaml

```diff
@@ -0,0 +1,31 @@
+friendly_name: Search Terms Daily
+description: |-
+  Daily count of terminal search terms collected by Merino.
+owners:
+  - ctroy@mozilla.com
+  - rburwei@mozilla.com
+workgroup_access:
+  - role: roles/bigquery.dataViewer
+    members:
+      - workgroup:search-terms/aggregated
+labels:
+  incremental: true
+bigquery:
+  time_partitioning:
+    field: submission_date
+    type: day
+    require_partition_filter: true
+    expiration_days: null
+scheduling:
+  dag_name: bqetl_search_terms_daily
+  arguments: ['--schema_update_option=ALLOW_FIELD_ADDITION']
+fields:
```

Comment:
`schema.yaml` should its own file: https://mozilla.github.io/bigquery-etl/cookbooks/creating_a_derived_dataset/#creating-the-table-schema. It's needed to run step 13 of https://mozilla.github.io/bigquery-etl/cookbooks/common_workflows/.